### PR TITLE
Feat/badge creation mobile

### PR DESCRIPTION
**Pull request by jona159**

### CODE REVIEW

File: src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.css [hunk 1]
```diff
@@ -1,11 +1,28 @@
 .heading {
-    padding-left: 1em; 
-    padding-right: 1em; 
-    padding-top: 1em;
+    padding: 0.25rem; 
     font-size: 28px;
+	font-size: 1.25rem/* 20px */;
```
Comment (YouQam) [inline src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.css:4]: please, delete one of the font-size.

File: src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.css [hunk 2]
```diff
@@ -14,12 +31,77 @@
...
 li::marker {
 	color: var(--color-blue);
 }
+
+.ng-autocomplete {
+	width:100%;
+	max-width: 500px;
+  }
+
+.competencyDuration {
```
Comment (YouQam) [inline src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.css:77]: This class has some duplicate attributes `font-family`, `line-height`...

**Review by YouQam — CHANGES_REQUESTED** [review]
The mobile view looks great, but there’s something needs to be updated.
When you choose  `Kompetenz` first then change it to `Teilnahme` the  `Badge erstellen`  button doesn’t work. I found that it asks for `Händisch Kompetenzen hinzufügen` mandatory fields although this section is hidden in `Teilnahme` category.

File: src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.css (hunk 1 above)
Comment (jona159) [reply src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.css]: Deleted the duplicates

File: src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.css (hunk 2 above)
Comment (jona159) [reply src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.css:124]: Deleted the duplicates

**Review by YouQam — APPROVED** [review]
Well done!

**Pull request merged**